contrib/src/uuid.rs

```diff
@@ -0,0 +1,117 @@
+extern crate uuid as uuid_ext;
+
+use std::fmt;
+use std::str::FromStr;
+use std::ops::{Deref, DerefMut};
+use rocket::request::FromParam;
+
+
+/// The UUID type, which implements `FromParam`. This type allows you to accept
+/// values from the [Uuid](https://github.com/rust-lang-nursery/uuid) crate in 
+/// your request handlers.
+///
+/// Make sure to enable the `uuid` feature in your Cargo.toml to use this type.
```

Comment:
Provide the TOML to do so, as with JSON. Also use the wording that the JSON/Template documentation uses. Something like, "The `uuid` feature must be enabled to...".